tests/test_slim_engine.py

```diff
@@ -421,6 +423,70 @@ def count_mut_types(ts):
             ).split()
             with pytest.raises(SystemExit):
                 capture_output(stdpopsim.cli.stdpopsim_main, cmd)
+        # test DFE with demography and interval as a bed file
```

Comment:
let's split this out into a separate `test_` function?

Reply:
I kind of think it is fine to have everything in one function, but lets just do this modification when we take this out of the test_slim_engine.py and transfer it to test_cli.py

Reply:
Ah I see what you mean, maybe later? :S